adagrams/game.py

```diff
@@ -1,11 +1,118 @@
+import random
+import string
 def draw_letters():
-    pass
+    hand_of_letters = []
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+    }
+    
+    #iterate through the dict to convert letter_pool to a list
+    LETTER_POOL_list = []  
```

Comment:
This variable name mixes upper and lower case. I would call this variable `letter_pool_list`. Technically it is not a constant because you create an empty list and then you add letters to it. 

Here's the official style guide for variable naming convention: https://peps.python.org/pep-0008/#function-and-variable-names

Reply:
Thank you! That makes sense. I will pay more attention to the style.